tests/plugins/inlinetoolbar/positioning.js

```diff
@@ -0,0 +1,78 @@
+/* bender-tags: inlinetoolbar */
+/* bender-ckeditor-plugins: toolbar,link,inlinetoolbar */
+/* bender-include: ../balloonpanel/_helpers/tools.js */
+/* global balloonTestsTools */
+
+( function() {
+	'use strict';
+
+	var restore = CKEDITOR.dom.window.prototype.getViewPaneSize;
+	CKEDITOR.dom.window.prototype.getViewPaneSize = function() {
```

Comment:
Can't it be done in before callback using `sinon.stub` and restored in after callback?